### ISSUE TITLE
Jellyfin preventing another application from starting properly

### ISSUE THREAD

**Issue by Kreeblah** [jellyfin/jellyfin#2188]
**Describe the bug**
I recently started looking into Jellyfin in order to see whether it might be able to replace Plex in my setup.  However, what I found is that if Jellyfin is running on my system, it's preventing Roon from starting up properly.  I created a thread on the Roon support forum at https://community.roonlabs.com/t/clients-suddenly-unable-to-connect-to-server/87870, as the behavior I was seeing in their application started a couple weeks after installing Jellyfin, but I guess I hadn't tried restarting Roon (which seems to be what triggers it).

I don't know whether this is something your team wants to look into, but I wanted to report it here as well, just in case.

**To Reproduce**
1. Install Roon (creating a trial account if needed) on Ubuntu 18.04
2. Set Roon up to use a non-root user
3. Set Jellyfin up to use the same non-root user
4. Start Jellyfin and wait for it to finish loading
5. Start Roon and watch /var/roon/RoonServer/Logs/RoonServer_log.txt

**Expected behavior**
Both applications should be able to run together.  However, Roon keeps restarting.  Jellyfin, on the other hand, seems to run fine.

**System (please complete the following information):**
 - OS: Ubuntu 18.04
 - Browser: N/A
 - Jellyfin Version: 10.4.3
 - Installed Plugins: Anime
 - Reverse proxy: nginx

**Comment by Kreeblah** [comment]
Oh, I should probably mention that I did not observe this behavior with Emby last I tried it, but it's been a while.  If it's useful, I can see whether the current Emby version does the same thing.

**Comment by Artiume** [comment]
So Roon uses ffmpeg, and perhaps upnp. Those are the only two things I can think of that might cause some sort of interference.

As an immediate solution, you could install jellyfin in docker to provide isolation.

**Comment by anthonylavado** [comment]
@Kreeblah If you tried Emby any time before approx. May 2018, then we should be clear.

I’ll see if I can spin this up later. Does Roon give any reason why it restarts? Is it a file in use issue or something?

**Comment by anthonylavado** [comment]
@Kreeblah I'd really love to try this to help you out, but I don't want to give Roon my credit card just to run a two week trial.

**Comment by anthonylavado** [comment]
I should have waited before installing (I've deleted my account anyway). 

There's this on their forums regarding Emby. I'm wondering if it's the same issue in Jellyfin: https://community.roonlabs.com/t/new-roonserver-install-with-restored-backup-in-repeated-crash-loop-if-you-try-to-login-from-a-client/81928/11

**Comment by Kreeblah** [comment]
Hey, sorry for letting this go so long.  The holidays were pretty hectic, and I'm backed up on a lot of things.

You actually don't need an account to test this issue at all.  I wasn't even getting to the point in Roon where I could enter credentials at all.  I'll give that a go with what you found regarding port 1900, though, and see whether I see the same behavior.

**Comment by Kreeblah** [comment]
I finally got around to trying this, so I wanted to post my results in case anybody else has the same issue.

Disabling all of the DLNA settings in Jellyfin does seem to resolve this for me.  I can have it installed and run with Roon, regardless of which one starts first.  I imagine I could have gotten by with just disabling the DLNA server piece, but I don't use DLNA for anything anyway, so I figured I might as well just turn it all off.

Thanks for finding that!
